libraries/pbrlib/genosl/mx_subsurface_bsdf.osl

```diff
@@ -1,6 +1,5 @@
 void mx_subsurface_bsdf(float weight, color _color, vector radius, float anisotropy, normal N, output BSDF bsdf)
 {
     // TODO: Subsurface closure is not supported by vanilla OSL.
-    bsdf.response = _color * weight * diffuse(N);
-    bsdf.throughput = color(0.0);
+    bsdf = _color * weight * diffuse(N);
```

Comment:
`testrender` will recognize the subsurface closure (even if it happens to use diffuse in the implementation right now). So it would be safe to use it.